app/Resources/views/page.html.twig

```diff
@@ -1,6 +1,9 @@
 {% extends '::base.html.twig' %}
 
 {% block head %}
+    {% if cb_id %}
```

Comment:
Does this need to both in the head and the content? Looking over this it seems to me like it's only needed in the content block like tag manager?

Reply:
I'm not sure why this was resolved. We can discuss this tomorrow.